lib/src/base-styles/_nx-forms.scss

```diff
@@ -10,10 +10,18 @@
 .nx-form {
   position: relative;
 
+  @include nx-container-helpers.container-vertical;
+
   &__required-field-asterisk {
     color: var(--nx-swatch-red-40);
   }
 
+  &__required-field-notice {
+    font-weight: 600;
+    color: var(--nx-color-text-dark);
+    margin-bottom: var(--nx-spacing-6x);
```

Comment:
This margin should be redundant now, I'm pretty certain it's what `.nx-p` already provides

Reply:
I checked to confirm and yes, the margin is the same, so it has been removed here: [4f092cd](https://github.com/sonatype/sonatype-react-shared-components/pull/735/commits/4f092cdf6a2317de6ef8934a45529ed4d7c2b40b)